gumby/modules/community_experiment_module.py

```diff
@@ -0,0 +1,58 @@
+from gumby.modules.experiment_module import ExperimentModule
+
+
+class CommunityExperimentModule(ExperimentModule):
+    def __init__(self, experiment, community_class):
+        super(CommunityExperimentModule, self).__init__(experiment)
+        self.community_class = community_class
+        self.community_launcher.finalize_callback = self._community_finalize
+
+    @property
+    def dispersy_provider(self):
+        from gumby.modules.base_dispersy_module import BaseDispersyModule
+
+        for module in self.experiment.experiment_modules:
+            if isinstance(module, BaseDispersyModule):
+                return module
+
+        self._logger.error("No dispersy provider module loaded. Load either DispersyModule or TriblerModule at the start of your scenario")
+        return None
+
+    @property
+    def dispersy(self):
+        return self.dispersy_provider.dispersy
+
+    @property
+    def session(self):
+        return self.dispersy_provider.session
+
+    @property
+    def session_config(self):
+        return self.dispersy_provider.session_config
+
+    @property
+    def community_loader(self):
+        return self.dispersy_provider.custom_community_loader
+
+    @property
+    def community_launcher(self):
+        return  self.community_loader.get_launcher(self.community_class.__name__)
+
+    @property
+    def community(self):
+        for c in self.dispersy.get_communities():
+            if isinstance(c, self.community_class):
```

Comment:
Yup, that's a pickle. Well sort of. It requires a bit of code, but I thought it best to leave that as future work, since it is not directly required for this PR. I added a todo comment here with the outline of how to handle that situation. The ChannelCommunity is only in gumby through the AllChannelModule. If future users want to create a ChannelCommunityModule, they run into this problem pretty quickly and read/pickup on this todo comment.